docs/ml-features.md

```diff
@@ -183,6 +183,78 @@ for words_label in wordsDataFrame.select("words", "label").take(3):
 </div>
 </div>
 
+## PolynomialExpansion
+
+[Polynomial expansion](http://en.wikipedia.org/wiki/Polynomial_expansion) is the process of expanding your features into a polynomial space, which is formulated by an n-degree combination of original dimensions. A [PolynomialExpansion](api/scala/index.html#org.apache.spark.ml.feature.PolynomialExpansion) class provides this functionality.  The example below shows how to expand your features into a 3-degree polynomial space.
+
+Note: We do not provide Python API for `PolynomialExpansion` in this version.
```

Comment:
Just realized: A Python API has been added.  Can you please added a Python example?  Thanks!